asciidoc/modules/con_mg_migrating-applications-to-vertx-http-client.adoc

```diff
@@ -201,3 +201,36 @@ response.end(ar -> {
   }
 });
 ----
++
+
+On the other hand, we can handle the response with some operations if we want to. You can see the difference in the next examples.
```

Comment:
instead of we use "you"

Reply:
@jcarranzan @vietj Can we rephrase this to:

You can also handle the response with operations. The following examples demonstrate handling responses with operations.

Reply:
ok I've updated it.
What do you think?

Thank you